subprojects/dependency-management/dependency-management.gradle

```diff
@@ -26,7 +26,6 @@ dependencies {
     compile libraries.gson
     compile libraries.jcip
```

Comment:
And here's a case where I think I'd prefer the "group:module" string, rather than an alias.